metrics/BUILD

```diff
@@ -25,6 +25,22 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+py_binary(
+    name = "bigquery",
+    srcs = ["bigquery.py"],
+    data = glob(["configs/**"]) + select({
+      ":k8": ["@jq_linux//file"],
```

Comment:
Should this be `k8s`?

Reply:
No, this is selecting over the --cpu flag in bazel, [k8](https://en.wikipedia.org/wiki/AMD_K8) is amd64 linux. in the future (0.8 which is in RC right now) there will be a much clearer way to select over platforms.

Reply:
Ack